src/main/java/hlf/java/rest/client/service/impl/UpdateChannelImpl.java

```diff
@@ -25,6 +27,12 @@
 @Service
 public class UpdateChannelImpl implements UpdateChannel {
 
+  private static final String VALUE_TAG_CAPABILITIES = "Capabilities";
+
+  private static final String FABRIC_2_0 = "V2_0";
+
+  private static final String DEFAULT_MOD_POLICY = "Admins";
```

Comment:
Let's define the constants in FabricClientConstants and re-use. I see these constants being defined in other places like ChannelServiceImpl, could be picked from one place.